app/javascript/components/components/Header.js

```diff
@@ -1,39 +1,101 @@
-import React, { Component } from 'react'
-import { Nav, NavItem } from 'reactstrap'
+import React, { useState } from 'react';
+import { Nav, NavItem, Navbar, NavbarToggler, NavLink, NavbarBrand, NavbarText, Collapse, DropdownToggle, UncontrolledDropdown, DropdownItem, DropdownMenu } from 'reactstrap'
 
-class Header extends Component {
-  render() {
-    const {
-      logged_in,
-      current_user,
-      new_user_route,
-      sign_in_route,
-      sign_out_route
-    } = this.props
-    console.log("logged_in:", logged_in)
-    console.log("current_user:", current_user)
-    return (
-      <>
-        <h1>React in Rails with Devise</h1>
-        <Nav>
+
+const Header = (props) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const toggle = () => {
+    setIsOpen(!isOpen);
+  };
+  const {
+    logged_in,
+    current_user,
+    new_user_route,
+    sign_in_route,
+    sign_out_route
+  } = props;
+  console.log("logged_in:", logged_in)
+  console.log("current_user:", current_user)
+  return (
+    <>
+
+      {/* <Nav>
           {logged_in &&
-            <NavItem>
+            <NavItem className='navBar' >
               <a href={sign_out_route} className="nav-link">Sign Out</a>
             </NavItem>
           }
           {!logged_in &&
-            <NavItem>
+            <NavItem className='navBar'>
               <a href={sign_in_route} className="nav-link">Sign In</a>
             </NavItem>
           }
           {!logged_in &&
-            <NavItem>
+            <NavItem className='navBar'>
               <a href={new_user_route} className="nav-link">Sign Up</a>
             </NavItem>
           }
-        </Nav>
-      </>
-    )
-  }
+        </Nav> */}
+
+      <div>
+        <Navbar
+          color="dark"
+          expand="md"
+          dark
+        >
+          <NavbarBrand href="/">
+            BIGBODYAPARTMENTS
+          </NavbarBrand>
+          <NavbarToggler onClick={toggle} />
+          <Collapse navbar isOpen={isOpen}>
+            <Nav
+              className="me-auto"
+              navbar
+            >
+              <NavItem>
+                <NavLink href="/ApartmentIndex/">
+                  VIEW APARTMENTS
+                </NavLink>
+              </NavItem>
+              <NavItem>
+                <NavLink href="/ApartmentNew">
+                  SAVED APARTMENTS
+                </NavLink>
+              </NavItem>
+              <UncontrolledDropdown
+                inNavbar
+                nav
+              >
+                <DropdownToggle
+                  caret
+                  nav
+                >
+                  MY APARTMENTS
+                </DropdownToggle>
+                <DropdownMenu end>
+                  <NavLink href="/ApartmentIndex/">
+                    ADD AN APARTMENT
+                  </NavLink>
```

Comment:
I would assume you would want to route to a different page for adding and editing apartments.